packages/atomic/src/components/facets/atomic-segmented-facet/atomic-segmented-facet.pcss

```
@@ -0,0 +1,39 @@
+@import '../../../global/global.pcss';
+@import '../facet-common.pcss';
+
+.box-container {
+  @apply flex items-stretch;
+}
+
+.box-container li:first-child .value-box:first-child {
+  border-radius: 3px 0px 0px 3px;
+}
+
+.box-container li:last-child .value-box:last-child {
+  border-radius: 0px 3px 3px 0px;
+}
+
+.value-box .value-label,
+.value-box-count {
+  @apply w-full;
+}
+
+.value-label {
```

Instead of using `!` which adds important (bad practice in css) we should use a more precise selector such as
```suggestion
.someparentclass .value-label {
```

---

Will fix !